pkg/tests/testinfra/testinfra.go

```diff
@@ -316,4 +322,5 @@ type GrafanaOpts struct {
 	DisableLegacyAlerting                 bool
 	EnableUnifiedAlerting                 bool
 	UnifiedAlertingDisabledOrgs           []int64
+	DisableRBAC                           bool
```

Comment:
Note: required for alerting tests with RBAC disabled, but if we manage to make all the tests run fine with RBAC (which would be great), we can remove this.